cronner/kronjob_util.py

```diff
@@ -19,12 +18,11 @@ def validate_kronjob_entry(entry):
         raise KronjobValidationException('entry names must not be longer than 52 chars. ({})'.format(entry['name']))
 
 
-def serialize_kronjob(base_template, entries):
+def serialize_kronjob(base_template, entries)
```

Comment:
hmm, is this valid syntax without the : ?

Reply:
😬